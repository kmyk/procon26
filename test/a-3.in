11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111100011111111111111111111
11111100001011111111111111111111
11111111111011111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111

1
00011100
11110100
00000100
00000000
00000000
00000000
00000000
00000000
