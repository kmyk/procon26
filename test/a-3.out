6 3 H 0
